File: 240623/특정 문자의 등장 횟수/number-appearances-of-a-particular-character.py
input_str = input()

firsts, seconds = 0, 0
for i in range(len(input_str) - 1):
    if input_str[i:i + 2] == "ee":
        firsts += 1
    elif input_str[i:i + 2] == "eb":
        seconds += 1

print(firsts, seconds)

File: 251204/특정 문자의 등장 횟수/number-appearances-of-a-particular-character.cpp
#include <iostream>
#include <string>
using namespace std;

int main() 
{
    string input_str;
    cin >> input_str;  

    int firsts = 0, seconds = 0;

    for (int i = 0; i < input_str.length() - 1; i++) 
    {
        if (input_str.substr(i, 2) == "ee") 
        {
            firsts++;
        }
        else if (input_str.substr(i, 2) == "eb") 
        {
            seconds++;
        }
    }

    cout << firsts << " " << seconds;

    return 0;
}
